cloud-init/mount-efs-share.tpl.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This is a Terraform template file, and the mount_point variable is
+# passed in via the templatefile().
+#
+# shellcheck disable=SC2154
+until findmnt "${mount_point}"
+do
+    sleep 2
+    echo "Attempting to mount EFS share."
```

Comment:
Is it worth echoing the value of the `mount_point` variable here to make any future troubleshooting easier?

Reply:
See commit 88ea9ce.